src/Engine/MapController.java

```diff
@@ -301,8 +301,8 @@ private void handlePathSelect(InputHandler.InputAction input)
         }
         break;
       case BACK:
-        myGameInputHandler.back();
         contemplatedAction.movePath = null;
+        myGameInputHandler.back();
```

Comment:
Moving this doesn't appear to change anything.

Reply:
I check if `movePath` is `null` in `onStateChange()`, and use that to determine whether (on a backwards input) I should put the cursor at the end of the movepath or on the acting unit.
Go figure.

Reply:
I still can't elicit a difference in behavior based on this code change Does it change anything for you?
While messing with this though, I did discover that if you select a unit, then deselect it, then go into the stats screen and back out; the cursor will center on the unit you selected, even though there is no reason it should.

Reply:
Get a DoR Carrier and put something in it.
Select Launch, and move the cursor around. Hit BACK.
With this swap, the cursor centers on the Carrier. Without it, it stays where it was (and renders the launch select menu there).

It's not even just the stats screen; just select/deselect, open the meta actions menu, and back out. What's happening is that when you *open* the meta actions menu, you hit `MapController` line 383 because your `GameInputState`'s holding onto the last unit you selected. You just don't see the result until you *close* the menu.
In any case, I just fixed it by pushing up a change to line 407, which *also* fixes the last unit you selected being hyper.

Reply:
Cool. That silly hyper-unit thing has been around for a while.
Although... If/when we add missiles to the game, I expect they will also use `FREE_TILE_SELECT`, so that's something to consider if that state assumes it's the default non-acting state.
I may be OK with leaving that problem to future us though.